atomics/T1090.001/T1090.001.yaml

```diff
@@ -18,17 +18,29 @@ atomic_tests:
     proxy_port:
       description: Proxy server port
       type: integer
-      default: 8080
+      default: 3128
     proxy_scheme:
       description: Protocol to proxy (http or https)
       type: string
       default: http
+  dependencies:
+  - description: Squid must be installed and running
+    prereq_command: if [ -x "$(command -v squid --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: |-
+      sudo DEBIAN_FRONTEND=noninteractive apt-get -y install squid curl
```

Comment:
Also this prereq command addresses only apt. Can you also add installations for redhat and macos here ?

Reply:
yep, added in [4a630fe](https://github.com/redcanaryco/atomic-red-team/pull/2963/commits/4a630fe1652c3c6f20e93619eac8e596a448d7cf)